crates/astria-sequencer/src/authority/state_ext.rs

```diff
@@ -140,7 +140,7 @@ impl<T: StateRead> StateReadExt for T {}
 
 #[async_trait]
 pub(crate) trait StateWriteExt: StateWrite {
-    #[instrument(skip(self))]
+    #[instrument(skip_all, fields(%address))]
```

Comment:
I'm not sure that we want to log the actual change being made here.

Seems noisy when tx lookup can give us the information?

Reply:
Removed in [1ceff06](https://github.com/astriaorg/astria/pull/1255/commits/1ceff067db6646e6b61f07a31b86635c70af0d2d).